### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Medical Records, Phase 1 Download Records

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#93120]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed feedback! Thank you so much for your patience.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Coding notes:** When selecting the types of medical records, there was some discussion around what would happen with a checkbox that selects all medical records. Checking now I'm not seeing that checkbox anymore --- but if you're still considering that route, this [article from Adrian Roselli](https://adrianroselli.com/2021/10/switch-role-support.html) may be helpful. I'll also suggest that you could break the interaction into two pages, starting with a yes/no radio button asking if you want all records or to choose the records you want, and then conditionally the next page with the checkboxes.
- [ ] **Should:** When giving people a "What to know before you download" additional info component, make sure it's positioned before they make a choice (eg. file format) about what they're downloading, just so they don't have to backtrack if they change their minds based on that information. Looking at the Figma file now, it looks like you've already moved things around and have made this change --- so thank you!
- [ ] **Consider:** I think it's worth exploring with stakeholders to see what other file formats users may want for their reports besides PDF and text. Maybe that's all? But in general I think it's a good practice to provide maximum flexibility to users to receive information in the format of their choice.
- [ ] **Must:** For the custom date range, make sure you're using the design system's date component. 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
